gpt-subtrans.py

```diff
@@ -10,6 +11,7 @@
 logging.basicConfig(
     format='%(levelname)s: %(message)s', 
     level=logging_level,
+    encoding='utf-8',
```

Comment:
It should only be necessary to configure the logger once, as it uses a global instance, so I think adding the encoding here and in gui-subtrans.py should be sufficient.

Reply:
okay